src/hip/CUDACore/radixSort.h

```diff
@@ -148,6 +150,7 @@ __device__ __forceinline__ void radixSortImpl(
 
     // broadcast
     ibs = size - 1;
+    ibs3 = ibs;
```

Comment:
could you add comments on the code about why the ibs2/ibs3 workaround is needed ?

Reply:
where ibs3 is used?